vpr/src/route/connection_router.cpp

```diff
@@ -1,4 +1,5 @@
 #include "connection_router.h"
+#include <algorithm>
```

Comment:
Ah! But it is best practice to put the header file of the file being implemented first, and then follow your proposed style after:

```
<header_for_file>
<STL headers>
<VPR specific headers>
```

I tried looking throughout the route folder, but somehow none of them include from the standard library... thats kinda insane... so I was unable to find "what everyone else does" lol.

I added a return carriage, since at least one file did something similar, to make things more clear:
```
<header_for_file>

<STL headers>
<VPR specific headers>
```